lib/wallet/src/Cardano/Wallet/Write/Tx/Balance/TokenBundleSizeAssessor.hs

```diff
@@ -0,0 +1,112 @@
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE TypeApplications #-}
+
+-- | Assessing sizes of token bundles
+module Cardano.Wallet.Write.Tx.Balance.TokenBundleSizeAssessor
```

Comment:
Can we shorten the name to `TokenBundleSize`?

Justification:
- It's more concise
- This module includes a greater variety of things than just the `TokenBundleSizeAssessor` type. For example:
    - `TokenBundleMaxSize`: defines the maximum size of a token bundle
    - `computeTokenBundleSerializedLengthBytes`: computes the serialised length of a token bundle